molecule/default/tests/test_verify_mbu_installed.py

```diff
@@ -10,15 +16,41 @@
 backup_folder_on_container = '/var/backup'
 
 
-@pytest.mark.incremental
+# ======================================================================================================================
+# Helpers
+# ======================================================================================================================
+def run_on_container(command, container, run_on_host):
+    """Run the given command on the given container.
+
+    Args:
+        command (str): The bash command to run.
+        container (str): The container to run the command on.
+        run_on_host (testinfra.Host): Testinfra host object to execute the
+                                      wrapped command on.
+
+    Returns:
+        testinfra.CommandResult: Result of command execution.
+    """
+
+    pre_command = ("lxc-attach -n {} "
+                   "-- bash -c".format(container))
+    cmd = "{} '{}'".format(pre_command, command)
+    output = run_on_host.run(cmd)
+    assert output.rc == 0
+    return output
+
+
+# ======================================================================================================================
+# Test Cases
+# ======================================================================================================================
+@pytest.mark.test_id('406be222-cbee-11e8-ae3a-0025227c8120')
+@pytest.mark.jira('asc-262')
+@pytest.mark.test_case_with_steps()
```

Comment:
With the exception of this addition, are the other changes in this file relevant to using steps or should they be broken out into a separate commit?

Reply:
@johnduarte Split PR into multiple commits to address your feedback.